spras/allpairs.py

```diff
@@ -112,6 +112,8 @@ def parse_output(raw_pathway_file, standardized_pathway_file):
         @param standardized_pathway_file: the same pathway written in the universal format
         """
         df = pd.read_csv(raw_pathway_file, sep='\t', header=None)
+
         df['Rank'] = 1  # add a rank column of 1s since the edges are not ranked.
         df = reinsert_direction_col_undirected(df)
-        df.to_csv(standardized_pathway_file, header=False, index=False, sep='\t')
+        df.columns = ['Node1', 'Node2', 'Rank', 'Direction']
```

Comment:
I'm trying to think about where to document that this header row is now part of the canonical pathway output. We don't have that type of user documentation anywhere yet. In the meantime, we can update the parts of the contributing guide that refer to `parse_output`.